sources/platform/integrations/actors/integration_ready_actors.md

```diff
@@ -69,9 +69,19 @@ It's also possible to combine both approaches, which is useful for development p
 
 ```js
 const { payload, datasetId } = await Actor.getInput();
-const datasetIdToProcess = payload?.resource?.defaultDatasetId ?? datasetId;
+const datasetIdToProcess = datasetId || payload?.resource?.defaultDatasetId;
 ```
 
+To make the integration process smoother it's possible to define an input that's going to be prefilled when actor is being used as an integration. You can do that in the "Settings" tab of your Actor, on the "Integrations" form. In our example, we'd use:
```

Comment:
To make the integration process smoother, it's possible to define an input that's going to be prefilled when Actor

Suggestion: 
-> adding comma after smoother
-> actor -> Actor

Reply:
Fixed